domains/Discussions/Tests/Feature/QuestionsStoreTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Domains\Discussions\Tests\Feature;
+
+use Domains\Accounts\Database\Factories\UserFactory;
+use Domains\Accounts\Models\User;
+use Domains\Discussions\Database\Factories\QuestionFactory;
+use Domains\Discussions\Models\Question;
+use Faker\Factory;
+use Faker\Generator;
+use Illuminate\Http\Response;
+use Illuminate\Support\Str;
+use Laravel\Lumen\Testing\DatabaseMigrations;
+use Tests\TestCase;
+
+class QuestionsStoreTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    private Generator $faker;
+    private User $user;
+    private Question $question;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->faker = Factory::create();
+        $this->user = UserFactory::new()->create();
+        $this->question = QuestionFactory::new(['author_id' => $this->user->id])->create();
+    }
+
+    /** @test */
+    public function it_stores_questions(): void
+    {
+        $payload = [
+            'title' => $this->faker->title,
+            'description' => $this->faker->paragraph,
+        ];
+
+        $response = $this->actingAs($this->user)
+            ->call('POST', route('discussions.questions.store'), $payload);
+
+        self::assertTrue($response->isEmpty());
```

Comment:
Care to check the response status code is the correct one? It should be 204.

Reply:
done